roaring/src/bitmap/statistics.rs

```diff
@@ -0,0 +1,106 @@
+use core::mem;
+
+use crate::bitmap::container::Container;
+use crate::RoaringBitmap;
+
+use super::store::Store;
+
+/// Detailed statistics on the composition of a bitmap.
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub struct Statistics {
```

Comment:
Probably worth a `#[non_exhaustive]`, to be able to add fields backwards-compatibly.